### DISCUSSION TITLE
stat-computing-r-python/part-wrangling/02c-good-graphics

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#43] [General]
# stat-computing-r-python/part-wrangling/02c-good-graphics

https://srvanderplas.github.io/stat-computing-r-python/part-wrangling/02c-good-graphics.html

**Comment by maksudatoma** [comment]
The Python plot for the life expectancy of four countries is not visible in section "22.1.4."
